src/tests/nurse/test_views.py

```diff
@@ -127,6 +145,92 @@ def s3_mock():
 }
 
 
+@pytest.fixture
+def prescription(user):
+    patient = Patient.objects.create(**patient_data)
+    nurse, _ = Nurse.objects.get_or_create(user=user)
+    nurse.patients.add(patient)
+    return Prescription.objects.create(patient=patient, **prescription_data)
+
+
+@pytest.mark.django_db
+class TestSendEmailToDoctorView:
+    def test_send_email_to_doctor(self, client, prescription):
+        url = reverse_lazy("send-email-to-doctor", kwargs={"pk": prescription.id})
+
+        response = client.post(url)
+        assert response.status_code == status.HTTP_200_OK
+        assert response.data == {"message": "Email sent"}
+        assert len(mail.outbox) == 1
+        assert mail.outbox[0].to == [prescription.email_doctor]
+
+    def test_send_email_to_doctor_404(self, client):
```

Comment:
Keep a consistent approach with the other tests for reusing url and testing endpoint, so you don't have to repeat yourself on the reverse_lazy() and we also know what the URL looks like
```python
    url = reverse_lazy("send-email-to-doctor")
    data = prescription_data

    def test_endpoint(self):
        assert self.url == "/whatever-it-is/"
```

Reply:
fixed